frontend/src/components/viewers/Tensorboard.test.tsx

```diff
@@ -102,9 +103,14 @@ describe('Tensorboard', () => {
     const config = { type: PlotType.TENSORBOARD, url: 'http://test/url' };
     const getAppMock = () => Promise.resolve({ podAddress: 'test/address', tfVersion: '1.14.0' });
     jest.spyOn(Apis, 'getTensorboardApp').mockImplementation(getAppMock);
+    jest.spyOn(Apis, 'isTensorboardPodReady').mockImplementation(() => Promise.resolve(true));
     tree = shallow(<TensorboardViewer configs={[config]} />);
 
     await TestUtils.flushPromises();
+    jest.runOnlyPendingTimers();
+    await TestUtils.flushPromises();
+    expect(setInterval).toHaveBeenCalledTimes(1);
+    expect(setInterval).toHaveBeenCalledWith(expect.any(Function), 5000);
```

Comment:
it feels to me `setInterval` doesn't align with this test. It just tests tree end result is enough.

Reply:
You mean removing the expectations on the function calls?